librosa/core/audio.py

```diff
@@ -332,6 +332,8 @@ def get_duration(y=None, sr=22050, S=None, n_fft=2048, hop_length=512,
     S : np.ndarray [shape=(d, t)] or None
         STFT matrix, or any STFT-derived matrix (e.g., chromagram
         or mel spectrogram).
+        The duration returned using this option will only be an
+        estimated value, reconstructed using the integer frames
```

Comment:
"reconstructed using the integer frames" might not be totally clear to a reader.

Suggested rephrase: "Durations calculated from spectrogram inputs are only accurate up to the frame resolution.  If high precision is required, it is better to use the audio time series directly."

... or something to that effect.